src/nl/hannahsten/texifyidea/lang/commands/LatexListingCommand.kt

```diff
@@ -19,6 +20,11 @@ enum class LatexListingCommand(
     LSTINPUTLISTING("lstinputlisting", "options".asOptional(), RequiredFileArgument("filename", false, commaSeparatesArguments = false), dependency = LISTINGS),
     DIRECTLUA("directlua", dependency = LUACODE),
     LUAEXEC("luaexec", dependency = LUACODE),
+    PY("py", dependency = PYTHONTEX),
+    PYB("pyb", dependency = PYTHONTEX),
+    PYC("pyc", dependency = PYTHONTEX),
+    PYS("pys", dependency = PYTHONTEX),
+    PYV("pyv", dependency = PYTHONTEX),
```

Comment:
If you add them here, you don't need to add them to LatexGenericRegularCommand

Reply:
I have removed the redundant definitions in `LatexGenericRegularCommand`. Just as a note, it appears `LUAEXEC` and `DIRECTLUA` are defined in both. Based on your comment, it sounded like you were suggesting `LatexGenericRegularCommand` is the duplication that should be removed. However, it looks like the `DIRECTLUA` and `LUAEXEC` that are in use are the ones from `LatexGenericRegularCommand`. Please let me know if you'd prefer that I switch around which ones are kept.

Reply:
Well spotted, thanks! I have removed those other duplicates as well